Compile error for USE_LED_STRIP and F3

Compile error for F3 fc with added USE_LED_STRIP (tested on OMNIBUS and FURYF3)
I get this error after the pull of the latest 4 merges.

```./src/main/interface/settings.c:1072:86: error: 'TABLE_RGB_GRB' undeclared here (not in a function); did you mean 'TABLE_DEBUG'?
     { "ledstrip_grb_rgb",           VAR_UINT8  | MASTER_VALUE | MODE_LOOKUP, .config.lookup = { TABLE_RGB_GRB }, PG_LED_STRIP_CONFIG, offsetof(ledStripConfig_t, ledstrip_grb_rgb) },
                                                                                      ^~~~~~~~~~~~~
                                                                                      TABLE_DEBUG
```
Removing the #define USE_LED_STRIP solves it

--- a/src/main/interface/settings.c
+++ b/src/main/interface/settings.c
@@ -341,7 +341,7 @@ static const char * const lookupOverclock[] = {
 };
 #endif
 
-#ifdef USE_LED_STRIP_STATUS_MODE
+#ifdef USE_LED_STRIP
     static const char * const lookupLedStripFormatRGB[] = {
         "GRB", "RGB"
     };
@@ -522,7 +522,7 @@ const lookupTableEntry_t lookupTables[] = {
 #ifdef USE_OVERCLOCK
     LOOKUP_TABLE_ENTRY(lookupOverclock),
 #endif
-#ifdef USE_LED_STRIP_STATUS_MODE
+#ifdef USE_LED_STRIP
     LOOKUP_TABLE_ENTRY(lookupLedStripFormatRGB),
 #endif
 #ifdef USE_MULTI_GYRO

--- a/src/main/interface/settings.h
+++ b/src/main/interface/settings.h
@@ -92,7 +92,7 @@ typedef enum {
 #ifdef USE_OVERCLOCK
     TABLE_OVERCLOCK,
 #endif
-#ifdef USE_LED_STRIP_STATUS_MODE
+#ifdef USE_LED_STRIP
     TABLE_RGB_GRB,
 #endif
 #ifdef USE_MULTI_GYRO

